src/transformers/models/auto/modeling_auto.py

```diff
@@ -428,6 +429,13 @@
     ]
 )
 
+MODEL_FOR_UNIVERSAL_SEGMENTATION_MAPPING_NAMES = OrderedDict(
```

Comment:
Wondering if we can add OneFormer to the MODEL_FOR_INSTANCE_SEGMENTATION_MAPPING_NAMES?

cc @alaradirik

Reply:
@NielsRogge absolutely, the pipeline calls the post-processing method anyway, it'd be easy to add OneFormer.

Reply:
Closing this as we'll keep it like this